backend/docs/developers/setup-local-env.rst

```diff
@@ -43,22 +45,58 @@ You need the following libraries and/or programs:
        npm run build
 
 #. Run the migrations with ``src/manage.py migrate``
-#. Create a superuser to access the management interface: ``src/manage.py createsuperuser``
-#. To set environment variables settings, vreate a ``.env`` file. You can use and modify the provided example:
 
-  .. code:: bash
+   .. note::
 
-    cp dotenv.example .env
+      If you get an error about the ``django.contrib.gis`` module,
+      you need to install the GDAL library and/or enable the extension or give superuser privileges. See the requirements in the Dockerfile.
+
+#. Create a superuser to access the management interface:
+
+   .. code:: bash
+
+      src/manage.py createsuperuser
+
+#. To set environment variables settings, create a ``.env`` file. You can use and modify the provided example:
+
+   .. code:: bash
+
+      cp dotenv.example .env
+
+#. Generate the translation files:
+
+   .. code:: bash
+
+      ./bin/make_translations
+      src/manage.py makemessages -a
```

Comment:
This is not needed (it is done inside the `make_translations.sh` script). Also, the .sh extension is missing in the line above.